source/app/datamgmt/manage/manage_webhooks_db.py

```
@@ -0,0 +1,277 @@
+
+
+def delete_webhook_by_id(webhook_id: int):
+    """Delete a webhook
+
+    Args:
+        webhook_id (int): webhook id
+    """
+    Webhook.query.filter_by(id=webhook_id).delete()
+
+
+import jsonschema
```

Ruff (0.8.2)

Redefinition of unused `jsonschema` from line 18

Remove definition: `jsonschema`

(F811)